core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2183,6 +2183,13 @@ object SparkContext extends Logging {
   }
 
   /**
+   * Return SparkContext
+   */
+  private[spark] def get(): SparkContext = {
```

Comment:
I think it's best to just remove this get method. We should have the upper layer (i.e. sparkcontext) send an explicit signal via function calls to the lower layer (e.g. rpc).

@zsxwing might know more about whether this is needed at all.